<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserPanel</class>
 <widget class="QMainWindow" name="UserPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QuestionnaireApp - userType</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="3,97">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
          <item>
           <widget class="QLabel" name="welcomeLabel">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Welcome firstName lastName!</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCommandLinkButton" name="logoutButton">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>logout</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="userTabWidget">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="administratorTab">
         <attribute name="title">
          <string>ADMINISTRATOR PANEL</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QCommandLinkButton" name="manageUsersButton">
              <property name="text">
               <string>Add / delete user accounts</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="teacherTab">
         <attribute name="title">
          <string>TEACHER PANEL</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QCommandLinkButton" name="questionBankButton">
              <property name="text">
               <string>Question bank</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCommandLinkButton" name="createTestButton">
              <property name="text">
               <string>Create test</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCommandLinkButton" name="seeTestResultsButton">
              <property name="text">
               <string>See candidates test results</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCommandLinkButton" name="manageCandidatesButton">
              <property name="text">
               <string>Manage candidates accounts</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="candidateTab">
         <attribute name="title">
          <string>CANDIDATE</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QLabel" name="availableTestTitleLabel">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Available tests</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="infoCandidateLabel">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Double click to start a test!</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="testTableWidget">
              <property name="styleSheet">
               <string notr="true">QTableWidget
{
	font: 15px &quot;Times New Roman&quot;;
}

QHeaderView::section 
{
    background-color: silver;
    border: none;
	font-family: Times new Roman;
	font-size: 16px;
}

QTableWidget::item:selected
{ 
background-color: rgb(36, 36, 36);
color: white;
}</string>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <property name="gridStyle">
               <enum>Qt::DashLine</enum>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
              <attribute name="horizontalHeaderCascadingSectionResizes">
               <bool>true</bool>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
